### ISSUE TITLE
Reproducing results from scratch not working

### ISSUE THREAD

**Issue by dreichCSL** [wenhuchen/Meta-Module-Network#11]
I want to train this model with new visual features from a different object detector. 

I've now trained this model from scratch using bootstrapping with all train data, as described. I'm using the program files that were provided in this repo. The only (major) difference is that I'm using different visual features (quality should not be worse than bottomup-generated features). After training a few epochs (bootstrapping followed by finetuning), it's obvious that I'm nowhere close to reaching the numbers in the paper, only reaching accuracy of ~40% on  both the balanced val set and the testdev set. 

Any ideas what the problem could be? Is there any non-obvious tuning done wrt your visual features?

**Comment by dreichCSL** [comment]
The problem was normalization of bbox features (including adding two extra dimensions like in the provided model) I had added to the GQA_v2 dataloader class (akin to GQA_v1). Had added it inside the training loop, so it wasn't applied in testing. Now works fine.

**Comment by LemonQC** [comment]
> The problem was normalization of bbox features (including adding two extra dimensions like in the provided model) I had added to the GQA_v2 dataloader class (akin to GQA_v1). Had added it inside the training loop, so it wasn't applied in testing. Now works fine.

The memory is out of utilization when I used
